src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -68,4 +68,45 @@ databaseChangeLog:
                    type: int
                    constraints:
                      foreignKey: true
-                     nullable: false
\ No newline at end of file
+                     nullable: false
+
+  - changeSet:
+      id: 2
+      author: krzysztof-jezewski
+      changes:
+        - addColumn:
+            tableName: users
+            columns:
+              - column:
+                  name: email
+                  type: varchar(50)
+                  constraints:
+                    nullable: false
+        - modifyDataType:
+            columnName: author_id
+            newDataType: char
+            tableName: tickets
+        - modifyDataType:
+            columnName: developer_id
+            newDataType: char
+            tableName: tickets
```

Comment:
It's necessary because you cannot cast int4 to uuid. I could either first change the type to char, and then to uuid or drop the column and add it again with correct type.

Reply:
Drop column and add it again seems like a better idea instead of type conversion.

Reply:
changed to drop column